decorator.rb

```diff
@@ -0,0 +1,11 @@
+# This class acts as a base decorator for Nameable objects. It encapsulates the Nameable object
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
```

Comment:
- [ ] Good job so far but here Getters and Setters for @nameable instance variables should be present so kindly add them to att_accessor with the date ✨🙏🏻